.yarnrc.yml

```diff
@@ -1,10 +1,10 @@
+compressionLevel: mixed
+
+enableGlobalCache: false
```

Comment:
Why?
You not be only for the CI so be part of the `setup` workflow?

Reply:
Oups I didn't catch this one, it was done by the yarn set

Reply:
This was the previous configuration on yarn3 and changed to true in yarn4 - this part was done by their migration script.